tests/unit/lambdas/meta.test.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-env jest */
+import MetaUtil from '../../../src/functions/oss/meta'
+
+import ossUtils from '../../../src/functions/oss/utils'
+ossUtils.updateEntry = jest.fn().mockImplementation(() => Promise.resolve(null))
+
+process.env.LAMBDA_ENV = 'production'
```

Comment:
Can you move this into the `beforeEach` then delete it in the `afterEach` or `afterAll`. My concern is there may be some side-effects to other tests if we leave it in here.